extensions/hawkbit-extension-artifact-repository-s3/pom.xml

```diff
@@ -0,0 +1,78 @@
+<!--
+
+    Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+   <parent>
+      <groupId>org.eclipse.hawkbit</groupId>
+      <artifactId>hawkbit-extensions-parent</artifactId>
+      <version>0.2.0-SNAPSHOT</version>
+   </parent>
+   <artifactId>hawkbit-extension-artifact-repository-s3</artifactId>
+   <name>hawkBit :: S3 Repository</name>
+
+
+   <dependencies>
+      <dependency>
```

Comment:
This is only working because `hawkbit-parant` is fortuitous importing `spring-cloud-dependencies` which provides  the needed dependency: 
```
<dependency>
    <groupId>org.springframework.cloud</groupId>
    <artifactId>spring-cloud-aws-dependencies</artifactId>
    <version>${spring-cloud-aws.version}</version>
    <type>pom</type>
    <scope>import</scope>
</dependency>
```
This is hard to figure out especially if you change dependencies in `hawkbit-parant` => there is only one `hawkbit-extensions-parent` to refer to a version of HawkBit. So all extensions are heavily depending on a certain version.